includes/languages/english.php

```diff
@@ -632,12 +632,7 @@ function zen_date_raw($date, $reverse = false) {
   $file_list = array(FILENAME_EMAIL_EXTRAS, FILENAME_HEADER, FILENAME_BUTTON_NAMES, FILENAME_ICON_NAMES, FILENAME_OTHER_IMAGES_NAMES, FILENAME_CREDIT_CARDS, FILENAME_WHOS_ONLINE, FILENAME_META_TAGS); 
   foreach ($file_list as $file) { 
     $file = str_replace(".php","",$file); 
-    if (file_exists(DIR_WS_LANGUAGES . $_SESSION['language'] . '/' . $template_dir . '/' . $file . '.php')) {
-      $template_dir_select = $template_dir . '/';
-    } else {
-      $template_dir_select = '';
-    }
-    require_once DIR_WS_LANGUAGES . $_SESSION['language'] . '/' . $template_dir_select . $file . '.php'; 
+    require_once(zen_get_file_directory(DIR_FS_CATALOG . DIR_WS_LANGUAGES . $_SESSION['language'] . "/", $file . ".php", 'false'));
```

Comment:
Why did you add the `. "/"`?  And why the double-quotes around `".php"`?

Reply:
You do need the '/' because $_SESSION['language'] doesn't have a trailing slash.  This is different from your finding above, which was a bug.